include/xsimd/types/xsimd_utils.hpp

```diff
@@ -196,6 +196,34 @@ namespace xsimd
         return res;
     }
 
+    namespace kernel
+    {
+        namespace detail
+        {
+            /**************************************
+             * enabling / disabling metafunctions *
+             **************************************/
+
```

Comment:
we should probably define a `void_t` type here, and use it for all the following `enable_..._t` types

Reply:
Haven't added it because I can't fathom yet how `void_t` works 😅